bittide-tools/clockcontrol/plot/Main.hs

```diff
@@ -224,13 +232,24 @@ instance Exception CsvParseError
 
 -- | Data post processor.
 postProcess ::
-  forall n k c m .
-  (KnownNat n, KnownNat k, KnownNat c, Monad m, c <= k) =>
-  ConduitT (Capture n c) (DataPoint n k) m ()
-postProcess = scanlC process initDummy
+  forall n k n' k' m.
```

Comment:
Not a fan of `n'` and `k'`, the `'` is meaningless to me. Can you rename them such that they are descriptive? Could you also add documentation on what these are / why they differ?